--- origsrc/hunspell-1.2.7/src/parsers/Makefile.am	2007-11-16 02:50:27.000000000 -0600
+++ src/hunspell-1.2.7/src/parsers/Makefile.am	2008-08-17 20:14:04.454887100 -0500
@@ -1,4 +1,4 @@
-lib_LIBRARIES=libparsers.a
+noinst_LIBRARIES=libparsers.a
 libparsers_a_SOURCES=firstparser.cxx htmlparser.cxx \
 		      latexparser.cxx  manparser.cxx \
 		      textparser.cxx 
--- origsrc/hunspell-1.2.7/src/tools/Makefile.am	2008-08-17 20:21:03.244177700 -0500
+++ src/hunspell-1.2.7/src/tools/Makefile.am	2008-08-17 20:21:58.026479500 -0500
@@ -8,7 +8,7 @@
 
 hunspell_munch_SOURCES=munch.c
 hunspell_unmunch_SOURCES=unmunch.c
-include_HEADERS=munch.h unmunch.h
+pkginclude_HEADERS=munch.h unmunch.h
 
 hunspell_example_SOURCES=example.cxx
 hunspell_example_LDADD = ../hunspell/libhunspell-1.2.la
